it/apps/src/main/content/jcr_root/apps/forms-core-components-it/clientlibs/clientlib-it-base/.content.xml

```diff
@@ -2,4 +2,5 @@
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
     jcr:primaryType="cq:ClientLibraryFolder"
     allowProxy="{Boolean}true"
-    categories="[forms-it.base]"/>
+    categories="[forms-it.base]"
```

Comment:
how are we fixing it in the product?
I think we should include them in the page component.

Reply:
We don't ship any template in product, and these clientlibs should be referred via template policy. (same is with sites). We should not include anything in page component apart from ui.frontent module (that encapsulates af2-web-runtime). Rest all the core component libraries should be include via template page policies. This needs correction as well.